config/sidekiq.yml

```diff
@@ -4,14 +4,17 @@
 
 :schedule:
   expire_registration_state:
-    cron: "0 */15 * * * *"
+    every: 15m
```

Comment:
Ah that's cool, much prefer this syntax

Reply:
Yeah, I started to write an hourly schedule and then thought "wait, which field is that again"?